backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc

```diff
@@ -21,9 +21,9 @@ include::modules/oadp-supported-architecture.adoc[leveloffset=+1]
 
 OpenShift API for Data Protection (OADP) is platform neutral. The information that follows relates only to {ibm-power-name} and to {ibm-z-name}.
 
-* {oadp-short} 1.1.7 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.1.7 in terms of backup locations for these systems.
-* {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12, 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.2.3 in terms of backup locations for these systems.
-* {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.3.1 in terms of backup locations for these systems.
+* {oadp-short} {oadp-version-1-1} was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} {oadp-version-1-1} in terms of backup locations for these systems.
```

Comment:
OADP-1.1 is no longer supported, no need to add new comments regarding it.

Reply:
So comment out or leave with the new `oadp-version-1-1` attributes, please?

thanks